lib/routes/account.js

```diff
@@ -1439,6 +1439,7 @@ module.exports = (
             lastAccessTime: isA.number().min(0).required().allow(null),
             lastAccessTimeFormatted: isA.string().optional().allow(''),
             userAgent: isA.string().max(255).required().allow(''),
+            os: isA.string().max(255).allow('Unknown'),
```

Comment:
If we are set on returning `'Unknown'` as the default, isn't the explicit `allow` unnecessary here? I think the others only have it because Joi doesn't allow `null` and the empty string by default.